### PR TITLE
Release 3.3.0 post

### PR DESCRIPTION
**Pull request by mikaelrandy**
3.3.0 release announcement post.

### CODE REVIEW

File: _posts/2018-03-15-release-3-3-0.markdown [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+date: 2018-03-16 10:00:00
+author: "@mikaelrandy"
+categories: release
+---
+
+We are proud to announce the availability of atoum [3.3.0][milestone]!
+
+## Stats
+
+* 43 [commits][commits],
```
Comment (Hywan) [inline _posts/2018-03-15-release-3-3-0.markdown]: I think the seconds `[commits]` can be omitted.

File: _posts/2018-03-15-release-3-3-0.markdown [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+* 129 files changed,
+* 7 contributors,
+* 13 [PR closed][changelog].
+
+## What's new
+
+As we work on the next [major release], this release focus on stabilization and bug fixes, but there is also some new features.
+
+#### `string::notMatches` assertion
+The new `notMatches` assertion on `string` asserter allow to test that a string do not match a regexp.
```
Comment (Hywan) [inline _posts/2018-03-15-release-3-3-0.markdown]:
```diff
- do not
+ does not
```

Comment (Hywan) [reply _posts/2018-03-15-release-3-3-0.markdown]:
```diff
- regexp
+ regular expression
```

File: _posts/2018-03-15-release-3-3-0.markdown [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+#### `@os` annotation
+If you want to write OS specific tests, atoum now provide an `@os` assertion to check OS before launching test
+
+Here is an example :
```
Comment (Hywan) [inline _posts/2018-03-15-release-3-3-0.markdown]:
```diff
- example :
+ example:
```

File: _posts/2018-03-15-release-3-3-0.markdown [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        public function testEverywhereButLinuxAndDarwin()
+        {
+
+        }
+    }
+}
+```
+
+#### Dot report
+If you need a condensed output of atoum execution, we had a new Dot report who only display test execution progression, without any other details :
```
Comment (Hywan) [inline _posts/2018-03-15-release-3-3-0.markdown]:
```diff
- we had a new Dot report who only display
+ we have a new dot report that only displays
```

Comment (Hywan) [reply _posts/2018-03-15-release-3-3-0.markdown]:
```diff
- details :
+ details:
```

File: _posts/2018-03-15-release-3-3-0.markdown [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+```$ bin/atoum -udr -f tests/units/classes/test.php
```
Comment (Hywan) [inline _posts/2018-03-15-release-3-3-0.markdown]:
```diff
- -udr
+ --use-dot-report
```

Better to use long option name for the documentation :-).

File: _posts/2018-03-15-release-3-3-0.markdown [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+## Stats
+
+* 43 [commits][commits],
+* 129 files changed,
+* 7 contributors,
+* 13 [PR closed][changelog].
+
+## What's new
+
+As we work on the next [major release], this release focus on stabilization and bug fixes, but there is also some new features.
```
Comment (Hywan) [inline _posts/2018-03-15-release-3-3-0.markdown]:
```diff
- focus
+ focuses
```

Comment (Hywan) [reply _posts/2018-03-15-release-3-3-0.markdown]:
```diff
- some new features.
+ some new features listed below:
```

**Review by Hywan — CHANGES_REQUESTED** [review]
Thanks for the blog post!
Few minor changes required though.

File: _posts/2018-03-15-release-3-3-0.markdown [hunk 7]
```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Release 3.3.0"
+date: 2018-03-16 10:00:00
```
Comment (Grummfy) [inline _posts/2018-03-15-release-3-3-0.markdown]: I would publish monday (don't forget to change in the name of file)

File: _posts/2018-03-15-release-3-3-0.markdown [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+categories: release
+---
+
+We are proud to announce the availability of atoum [3.3.0][milestone]!
+
+## Stats
+
+* 43 [commits][commits],
+* 129 files changed,
+* 7 contributors,
```
Comment (Grummfy) [inline _posts/2018-03-15-release-3-3-0.markdown]: I got 6 contributors in my side ?

I think adding also the fact that we have also theses stats
* 7 [new features][changelog],
* 5 [bug fix][changelog].

File: _posts/2018-03-15-release-3-3-0.markdown [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+#### Dot report
+If you need a condensed output of atoum execution, we had a new Dot report who only display test execution progression, without any other details :
+
+```$ bin/atoum -udr -f tests/units/classes/test.php
+............................................................ [60/65]
+.....                                                        [65/65]
+
+Success (1 test, 65/65 methods, 0 void method, 0 skipped method, 872 assertions)!
+```
+
```
Comment (Grummfy) [inline _posts/2018-03-15-release-3-3-0.markdown:65]: I would add a link to the documentation, saying that the documentation has already been updated ;)

**Review by Grummfy — CHANGES_REQUESTED** [review]

**Comment by Grummfy** [comment]
thanks for the blog post. For information, if you need there is a helper for that type of blog post in the script directory.

File: _posts/2018-03-15-release-3-3-0.markdown (hunk 9 above)
Comment (mikaelrandy) [reply _posts/2018-03-15-release-3-3-0.markdown:65]: I think the documentation should be always up to date, and it should not be necessary to specify this point.

**Review by Grummfy — APPROVED** [review]

File: _posts/2018-03-15-release-3-3-0.markdown [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+#### Memory usage
+Memory usage information displayed after test suite execution now display real maximum memory usage.
+
+Previously, in case of garbage collector execution, memory usage value could be wrong.
```
Comment (Hywan) [inline _posts/2018-03-15-release-3-3-0.markdown:69]:
```diff
- could be wrong
+ was wrong
```

**Review by Hywan — CHANGES_REQUESTED** [review]
Minor typo. Then you can commit.

**Review by Hywan — APPROVED** [review]

**Pull request merged**